site-src/blog/2022/first-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+description: >
+  We're pleased to announce the start of beta releases for the Gateway API
+  project with the release of version `v0.5.0`. This release includes `v1beta1`
+  versions of `GatewayClass`, `Gateway` and `HTTPRoute` as well as several other
+  significant improvements which this post will cover some of the highlights of.
+---
+
+# Introducing Gateway API v1beta1
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: June 22, 2022 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We're pleased to announce the start of beta releases for the Gateway API
+project with the release of version `v0.5.0`. This release includes `v1beta1`
+versions of `GatewayClass`, `Gateway` and `HTTPRoute` as well as several other
+significant improvements which this post will cover some of the highlights of.
```

Comment:
I'm not sure this is better, but I think some tweaks would be good here since the opening paragraph seems so important for these kinds of posts.

```suggestion
We're pleased to announce that key Gateway API resources are graduating to
beta with the `v0.5.0` release of the API. In addition to `GatewayClass`, `Gateway` 
and `HTTPRoute` graduating to beta, this release introduces several new concepts
to the API that we'll cover below.
```